page-factory-core/src/main/java/ru/sbtqa/tag/stepdefs/CoreGenericSteps.java

```diff
@@ -139,4 +139,13 @@ public void userActionListParam(String action, List<String> list) throws NoSuchM
     public void isElementFocused(String element) {
         LOG.warn("Note that isElementFocused method is still an empty!");
     }
+
+    /**
+     * Current step will be replaced with steps of specified scenario
+     *
+     * @param fragmentName scenario name to insert instead of this step
+     */
+    public void userInsertsFragment(String fragmentName) {
+        LOG.info(String.format("This step will be replaced with scenario with name \"%s\"", fragmentName));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Invoke method(s) only conditionally. ([squid:S2629](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS2629))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=70&issues=AWSwZVrRIUupByjQG-e7&open=AWSwZVrRIUupByjQG-e7)

[ISSUE_KEY]: AWSwZVrRIUupByjQG-e7